packages/longhorn-crd/longhorn-1.4/package.yaml

```diff
@@ -0,0 +1,4 @@
+url: https://github.com/longhorn/charts.git
+subdirectory: charts/longhorn
+commit: 0a6375d9fdc89305ad08ae31e7942cefd72717c7
```

Comment:
Could you please provide how you chose this commit?

Reply:
This is because this commit fixes the invalid format of `question.yaml`, so use this commit as based for chart file generation.

https://github.com/longhorn/charts/commit/0a6375d9fdc89305ad08ae31e7942cefd72717c7

Reply:
I think what @PhanLe1010 asked is why did not use the 1.4.0 commit instead. 

Basically, Rancher chart is post-release after the upstream/community release, so if any issues we found before releasing it, we will have a chance to fix them at the last minute to avoid impacting user installation. In this case, there are two invalid indent issues found, so we need to include those fixes in this PR.

Reply:
Okay, I will use the the commit with 1.4.0 tag. And then do the subsequent fixes in Rancher chart PR. Thanks.

Reply:
U misunderstood it. I just explained what we use the commit after 1.4.0 commit. This is intended.

Reply:
Thanks @weizhe0422 and @innobead 
All clear to me